doc/source/whatsnew/v0.23.1.txt

```diff
@@ -78,7 +78,8 @@ Indexing
 
 - Bug in :meth:`Series.reset_index` where appropriate error was not raised with an invalid level name (:issue:`20925`)
 - Bug in :func:`interval_range` when ``start``/``periods`` or ``end``/``periods`` are specified with float ``start`` or ``end`` (:issue:`21161`)
--
+- Bug in :meth:`MultiIndex.set_names` where error raised for a MultiIndex with self.nlevels == 1 (:issue:`21149`)
```

Comment:
just say ``nlevels == 1`` (in double backticks)

Reply:
Thanks - done